src/VkBootstrap.cpp

```diff
@@ -844,7 +844,10 @@ std::vector<const char*> check_device_extension_support(
 }
 
 // clang-format off
-bool supports_features(VkPhysicalDeviceFeatures supported, VkPhysicalDeviceFeatures requested) {
+bool supports_features(VkPhysicalDeviceFeatures supported,
+                       VkPhysicalDeviceFeatures requested,
+                       const std::vector<ExtensionFeatures>& extension_supported,
```

Comment:
Gonna have to ask you to use east-const. (for consistency)
what that looks like.
There are multiple places this occurs fyi.
```cpp
std::vector<ExtensionFeatures> const& extension_supported,
```